core-bundle/src/DataContainer/DataContainerGlobalOperationsBuilder.php

```diff
@@ -0,0 +1,269 @@
+    ) {
+    }
+
+    public function __toString(): string
+    {
+        if (!$this->operations) {
+            return '';
+        }
+
+        return $this->twig->render('@Contao/backend/data_container/global_operations.html.twig', [
+            'operations' => $this->operations,
+        ]);
+    }
+
+    public function initialize(string $table): self
+    {
+        if (null !== $this->operations) {
+            throw new \RuntimeException(self::class.' has already been initialized.');
+        }
+
+        $builder = clone $this;
+        $builder->table = $table;
+        $builder->operations = [];
+
+        return $builder;
+    }
+
+    /**
+     * @param array{html: string}|array{
+     *     href: string,
+     *     title: string,
+     *     attributes?: HtmlAttributes,
+     * } $operation
+     */
+    public function prepend(array $operation): self
+    {
+        $this->ensureInitialized();
+
+        array_unshift($this->operations, $operation);
+
+        return $this;
+    }
+
+    /**
+     * @param array{html: string}|array{
+     *     href: string,
+     *     label: string,
+     *     title?: string,
+     *     attributes?: HtmlAttributes,
+     * } $operation
+     */
+    public function append(array $operation): self
+    {
+        $this->ensureInitialized();
+
+        $this->operations[] = $operation;
+
+        return $this;
+    }
+
+    public function addBackButton(string|null $href = null): self
+    {
+        $this->ensureInitialized();
+
+        if (null === $href) {
+            $href = $this->framework->getAdapter(System::class)->getReferer(true);
+        } elseif (str_contains($href, '=') && !str_contains($href, '?')) {
+            $href = $this->urlGenerator->generate('contao_backend').'?'.$href;
+        }
+
+        $this->append([
+            'href' => $href,
+            'label' => $GLOBALS['TL_LANG']['MSC']['backBT'],
+            'title' => $GLOBALS['TL_LANG']['MSC']['backBTTitle'],
+            'attributes' => (new HtmlAttributes())->addClass('header_back')->set('accesskey', 'b')->set('data-action', 'contao--scroll-offset#discard'),
+        ]);
+
+        return $this;
+    }
+
+    public function addClearClipboardButton(): self
+    {
+        $this->ensureInitialized();
+
+        $this->append([
+            'href' => $this->framework->getAdapter(Backend::class)->addToUrl('clipboard=1'),
+            'label' => $GLOBALS['TL_LANG']['MSC']['clearClipboard'],
+            'attributes' => (new HtmlAttributes())->addClass('header_clipboard')->set('accesskey', 'x'),
+        ]);
+
+        return $this;
+    }
+
+    public function addNewButton(string $href): self
+    {
+        $this->ensureInitialized();
+
+        $labelNew = $GLOBALS['TL_LANG'][$this->table]['new'] ?? $GLOBALS['TL_LANG']['DCA']['new'];
+
+        $this->append([
+            'href' => $href,
+            'label' => $labelNew[0] ?? '',
+            'title' => $labelNew[1] ?? '',
```

Comment:
```suggestion
            'label' => $labelNew[0] ?? null,
            'title' => $labelNew[1] ?? null,
```
Down the line, we do check if it the value is set but returning an empty string will make sure it always exists.
Could just return null here and let the twig default handle that properly, right?

Reply:
Improved in da583f1daf31379bd3b33accc4f5fdbda1c3b0c3